erpnext/payroll/report/total_payments_based_on_payment_mode/total_payments_based_on_payment_mode.json

```diff
@@ -0,0 +1,30 @@
+{
+ "add_total_row": 0,
+ "creation": "2020-06-02 01:04:41.926527",
+ "disable_prepared_report": 0,
+ "disabled": 0,
+ "docstatus": 0,
+ "doctype": "Report",
+ "idx": 0,
+ "is_standard": "Yes",
+ "modified": "2020-06-02 01:04:41.926527",
+ "modified_by": "Administrator",
+ "module": "Payroll",
+ "name": "Total Payments Based On Payment Mode",
```

Comment:
Rename to "Salary Payments Based On Payment Mode"

Reply:
done